collect_app/src/androidTest/java/org/odk/collect/android/SharedPreferencesTest.java

```diff
@@ -65,7 +66,7 @@ public void generalSharedPreferencesUpgradeTest() {
         GeneralSharedPreferences generalSharedPreferences = GeneralSharedPreferences.getInstance();
         for (String key : SharedPreferencesUtils.getAllGeneralKeys()) {
             if (key.equals(KEY_COMPLETED_DEFAULT)) {
-                assertEquals(generalSharedPreferences.get(key), false);
+                assertFalse((Boolean) generalSharedPreferences.get(key));
```

Comment:
These should cast to boolean, not Boolean. Since you must cast the Object, you might as well cast it to the primitive type that assertFalse expects.

Reply:
Thanks!